Lib/test/test_socket.py

```diff
@@ -2459,6 +2472,41 @@ def testCreateScoSocket(self):
             pass
 
 
+@unittest.skipUnless(HAVE_SOCKET_HYPERV,
+                     'Hyper-V sockets required for this test.')
+class BasicHyperVTest(unittest.TestCase):
```

Comment:
Yea unfortunately I’m not sure if I can actually test this in CI in anyway. It would require the tests to run on A Hyper-V enabled host. I just added tests for what I could and manually tested it myself. I know this isn’t ideal but it someone has another idea I can look into it.